File: PgAce.App/Forms/SqlEditorPane.cs
using ScintillaNET;
using WeifenLuo.WinFormsUI.Docking;
using System.Windows.Forms;

namespace PgAce.App.Forms
{
    public class SqlEditorPane : DockContent
    {
        public SqlEditorPane()
        {
            Text = "SQL Editor";
            var editor = new Scintilla
            {
                Dock = DockStyle.Fill,
            };
            Controls.Add(editor);
        }
    }
}

File: PgAce.App/Forms/SqlEditorPane.vb
Imports ScintillaNET
Imports WeifenLuo.WinFormsUI.Docking
Imports System.Windows.Forms

Namespace PgAce.App.Forms
    Public Class SqlEditorPane
        Inherits DockContent

        Public Sub New()
            Text = "SQL Editor"
            Dim editor = New Scintilla() With {
                .Dock = DockStyle.Fill
            }
            Controls.Add(editor)
        End Sub
    End Class
End Namespace
